platform_packages.md

```diff
@@ -54,27 +62,36 @@ go get github.com/odeke-em/drive/cmd/drive
 $GOPATH/bin/drive
 ```
 
-### Fedora
-Fedora rpms are available from the Fedora Copr project [here](https://copr.fedorainfracloud.org/coprs/vaughan/drive-google/)
-
-Enable the copr repository:
+---
+### Ubuntu, or Ubuntu based distros. 
+This [PPA](https://launchpad.net/~twodopeshaggy/+archive/ubuntu/drive) includes Ubuntu, Mint, Linux Lite, etc. [List](http://distrowatch.com/search.php?basedon=Ubuntu)
 
-```
-dnf copr enable vaughan/drive-google
+```sh
+sudo add-apt-repository ppa:twodopeshaggy/drive
+sudo apt-get update
+sudo apt-get install drive
 ```
 
-Install the package (drive-google):
+### Scripts for the latest stable Ubuntu distribution. 
+This [PPA](https://gitlab.com/jean-christophe-manciot/ppa) offers scripts for automated drive commands including syncing and many other goodies.
 
+```sh
+sudo sh -c $'echo "deb https://gitlab.com/jean-christophe-manciot/ppa/raw/master/Ubuntu yakkety stable #JC Manciot\'s Stable PPA" >> /etc/apt/sources.list.d/jean-christophe-manciot.list'
+sudo apt-get update
+sudo apt-get install drive-google
 ```
-dnf install drive-google
-```
 
+More details about the installation are available on the repository [home page] (https://gitlab.com/jean-christophe-manciot/Drive).
+
+---
 ### Packages Provided By
 
 Platform | Author |
 ---------| -------|
 [Arch Linux](https://aur.archlinux.org/packages/drive) | [Jonathan Jenkins](https://github.com/shaggytwodope)
-[Ubuntu Linux](https://launchpad.net/~twodopeshaggy/+archive/ubuntu/drive) | [Jonathan Jenkins](https://github.com/shaggytwodope)
+[Debian Linux](http://shaggytwodope.github.io/repo) | [Jonathan Jenkins](https://github.com/shaggytwodope)
+[Fedora Linux](https://copr.fedorainfracloud.org/coprs/vaughan/drive-google/) | [Vaughan Agrez](https://github.com/agrez)
 [openSUSE Linux]() | [Grant Rostig](https://github.com/grantrostig)
-[Fedora](https://copr.fedorainfracloud.org/coprs/vaughan/drive-google/) | [Vaughan Agrez](https://github.com/agrez)
+[Ubuntu Linux](https://launchpad.net/~twodopeshaggy/+archive/ubuntu/drive) | [Jonathan Jenkins](https://github.com/shaggytwodope)
+[Latest stable Ubuntu Linux](https://gitlab.com/jean-christophe-manciot/ppa) | [Jean-Christophe Manciot](https://gitlab.com/jean-christophe-manciot)
```

Comment:
- ```auto-sync``` would not be an accurate description since there is no daemon running.
- ```Latest stable``` is about Ubuntu version, not drive version. It means that only Ubuntu 16.10 yakkety is supported, although the scripts should work on earlier versions. Also, there is a current dev version (17.04) which is considered unstable. Hence the ```Latest stable Ubuntu```.
I can change this to ```Automation on Ubuntu Linux``` and people can get details on the home page of the PPA.